analytics.js

```diff
@@ -194,6 +206,8 @@ class Analytics {
         );
       } catch (e) {
         logger.error(e);
+        e.message = `[cookieConsent init failed]:: ${e.message}`;
+        notifyError(e);
```

Comment:
All the calls to `notifyError` can be replaced with `handleError` as it does the job of logging the error and notifying BugSnag.

Reply:
done